cmake/SetupCUDA.cmake

```diff
@@ -2,23 +2,39 @@
 # the list of CUDA libraries that we are using
 FUNCTION(SET_CUDA_LIBRARIES)
   ADD_DEFINITIONS(-DMFMG_WITH_CUDA)
+  FIND_LIBRARY(CUSPARSE cusparse PATH ${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES})
+  FIND_LIBRARY(CUSOLVER cusolver PATH ${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES})
+  MESSAGE("implicit ${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES}")
```

Comment:
Did you mean to leave that print statement?

Reply:
I just rebased and didn't touch anything yet.